src/types/venues.ts

```diff
@@ -178,6 +181,11 @@ export interface GenericVenue extends BaseVenue {
   template: GenericVenueTemplates;
 }
 
+export interface AnimateMapVenue extends BaseVenue {
+  id: string;
+  template: VenueTemplate.animatemap;
+}
```

Comment:
`id` shouldn't be baked directly into this type like this, as in a majority of cases, it's not guaranteed to be present based on how we query the data. We have a helper type (`WithId`) for showing this distinction where necessary.

```suggestion
export interface AnimateMapVenue extends BaseVenue {
  template: VenueTemplate.animatemap;
}
```

Reply:
Once the `id` key is removed, this is essentially just the same as `BaseVenue`. Is it anticipated that more specific data is going to be added here in future? If so, it's fine to keep this as is for now. But if not, then we should just use `GenericVenue`